From 6fc3a2e9103cb0d71f428f4192bb3ae5c7595cf0 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 30 Mar 2022 07:13:54 +0530
Subject: [PATCH] Remove usage of F-string

The usage of F-string is supported only from python 3.6 and
was giving "SyntaxError: invalid syntax" during build

Change-Id: I3b5e9dcc4dbf12dfd1459eb7d312517058f6e228
Signed-off-by: svenate <salini.venate@intel.com>
---
 tools/releasetools/common.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index c6ce724f75..b422f215e8 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -1665,8 +1665,7 @@ def _BuildBootableImage(image_name, sourcedir, fs_config_file, info_dict=None,
 
     BOOT_SIGNATURE_SIZE = 16 * 1024
     if len(boot_signature_bytes) > BOOT_SIGNATURE_SIZE:
-      raise ValueError(
-          f"GKI boot_signature size must be <= {BOOT_SIGNATURE_SIZE}")
+      raise ValueError('GKI boot_signature size must be <= {}'.format(BOOT_SIGNATURE_SIZE))
     boot_signature_bytes += (
         b'\0' * (BOOT_SIGNATURE_SIZE - len(boot_signature_bytes)))
     assert len(boot_signature_bytes) == BOOT_SIGNATURE_SIZE
-- 
2.17.1

